UsabillaSampleApp/www/css/index.css

```diff
@@ -16,106 +16,138 @@
-    width:100%;                   /* text area width */
-    text-align:center;
-    padding:180px 0px 0px 0px;     /* image height is 200px (bottom 20px are overlapped with text) */
-}
-
-/* Landscape layout (with min-width) */
-@media screen and (min-aspect-ratio: 1/1) and (min-width:400px) {
-    .app {
-        background-position:left center;
-        padding:75px 0px 75px 170px;  /* padding-top + padding-bottom + text area = image height */
-        margin:-90px 0px 0px -198px;  /* offset vertical: half of image height */
-                                      /* offset horizontal: half of image width and text area width */
-    }
-}
-
-h1 {
-    font-size:24px;
-    font-weight:normal;
-    margin:0px;
-    overflow:visible;
-    padding:0px;
-    text-align:center;
-}
-
-.event {
-    border-radius:4px;
-    -webkit-border-radius:4px;
-    color:#FFFFFF;
-    font-size:12px;
-    margin: 10px auto;
-    padding:2px 0px;
-    max-width: 200px;
-}
-
-.event.listening {
-    background-color:#333333;
-    display:block;
-}
-
-.event.received {
-    background-color:#4B946A;
-    display:none;
-}
-
-@keyframes fade {
-    from { opacity: 1.0; }
-    50% { opacity: 0.4; }
-    to { opacity: 1.0; }
-}
- 
-@-webkit-keyframes fade {
-    from { opacity: 1.0; }
-    50% { opacity: 0.4; }
-    to { opacity: 1.0; }
-}
- 
-.blink {
-    animation:fade 3000ms infinite;
-    -webkit-animation:fade 3000ms infinite;
-}
-
-.event.received.formDetails {
-    padding: 20px 20px;
-    display: inline-block !important;
-    max-width: 600px;
-    background-color: transparent;
-}
+  @font-face {
+    font-family: 'MiloOT-Bold';
+    src: url('../fonts/MiloOT-Bold.otf');
+  }
 
+  @font-face {
+    font-family: 'MiloOT-Medi';
+    src: url('../fonts/MiloOT-Medi.otf');
+  }
 
+ body {
+    background-image: url('../img/background.png');
+    background-size: 100%;
+  }
+  
+  .app {
+      /*text-align:center; */
+  }
+  
+  .ub-title {
+      height: 192px;
+      width: 272px;
+      color: #00A5C9;
+      font-family: "MiloOT-Bold";
+      font-size: 55px;
+      font-weight: bold;
+      letter-spacing: 0;
+      margin-left: 30px;
+      margin-top: 30px;
+      line-height: 65px;
+  }
+  
+  .ub-group {
+      text-align:center;
+      /* position: fixed; */
+      margin-top: 50%;
+      -webkit-transform: translateY(-50%);
+      -moz-transform: translateY(-50%);
+      -ms-transform: translateY(-50%);
+      -o-transform: translateY(-50%);
+      transform: translateY(-50%);
+  }
+  
+  .show-form {
+      /* text-align: center; */
+  }
+  .show-form .btn{
+      color: #FFFFFF;
+      font-family: "MiloOT-Medi";
+      font-size: 19px;
+      font-weight: 500;
+      letter-spacing: 0;
+      line-height: 19px;
+      border-radius: 2px;
+      background-color: #00A5C9;
+      width: 334px;
+      height: 48px;
+  }
+  
+  .event-group {
+      /* text-align: center; */
+  }
+  
+  .show-event {
+      /* text-align: center; */
```

Comment:
Can the comment be removed?

Reply:
It's kept for future purpose. but can be removed from scope and kept it somewhere with the comments